lib/connect/api/listingRequests/actions.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This file is part of the Ingram Micro Cloud Blue Connect Zapier Extension.
+ *
+ * @copyright (c) 2020 Ingram Micro, Inc. All Rights Reserved.
+ */
+
+const moment = require('moment');
+
+
+const { Listings } = require('@cloudblueconnect/connect-javascript-sdk');
+// const { filtersToQuery, paramDictToArray, paramErrorDictToArray } = require('../helpers/data');
+const { filtersToQuery } = require('../helpers/data');
+
+const LISTING_REQUEST_FILTERS_MAP = {
+  id: 'id',
+  marketplace_id: 'marketplace.id',
```

Comment:
The Listing Requests endpoint does not support RQL, please check the filter names.

For example to filter for marketplace id the filter field to send to the backend must be marketplace__id. (with double underscore)